tools/conf/pfPorts/poudriere_bulk

```diff
@@ -7,6 +7,7 @@ databases/php72-pdo_mysql
 databases/php72-pgsql
 devel/aws-sdk-php
 devel/gdb
+devel/hyperscan
```

Comment:
This line should be removed. Hyperscan is built as dependency of suricata package